lapack/gonum/dgetri.go

```diff
@@ -49,7 +49,7 @@ func (impl Implementation) Dgetri(n int, a []float64, lda int, ipiv []int, work
 	if nb > 1 && nb < n {
 		iws := max(ldwork*n, 1)
 		if lwork < iws {
-			nb = lwork / ldwork
+			nb = lwork / n
```

Comment:
This doesn't look like a row vs col major issue, so does this affect the NETLIB implementation as well?

Reply:
It is a row vs column major issue. This line calculates "how many columns fit into the workspace we have". For reference the column size is ldwork, for us it's n.

Reply:
Yep ... sorry.